config-connector/solutions/iam/kpt/service-account/Kptfile

```diff
@@ -0,0 +1,30 @@
+apiVersion: kpt.dev/v1alpha1
+kind: Kptfile
+metadata:
+  name: .
+packageMetadata:
+  shortDescription: sample description
+openAPI:
+  definitions:
+    io.k8s.cli.setters.service-account-name:
+      x-k8s-cli:
+        setter:
+          name: service-account-name
+          value: ${SERVICE_ACCOUNT?}
```

Comment:
This could be something like 'new-service-account'.

Reply:
Maybe "service-account-solution"? We probably want to pick a relatively unique name.

Reply:
Using "service-account-solution" as a default value now.